modules/Bio.js

```diff
@@ -0,0 +1,85 @@
+import Gallery from './Gallery.js'
+
+const Bio = () => `<div class="about">
```

Comment:
Let's indent this properly. I'll recommend 2 spaces per indent for your codebase.

You should also start your HTML code in a new line.